pontoon/base/static/js/theme-switcher.js

```diff
@@ -0,0 +1,79 @@
+$(function () {
+  function getSystemTheme() {
+    if (
+      window.matchMedia &&
+      window.matchMedia('(prefers-color-scheme: light)').matches
```

Comment:
We should change this back to dark (or switch the returns). Sorry for misleading you - the Auto setting on macos changes based on time and has no effect here.